ui/app/templates/releases.html

```diff
@@ -72,8 +72,8 @@ <h3 class="panel-title">
       <div style="float:right">
         <i ng-if="release_name && $first && (currentPage == 1)">Current</i>
         <button ng-if="release_name && ((currentPage != 1) || !$first )" class="btn btn-default btn-xs" ng-click="openRevertModal(release)">Revert to this</button>
-        <button ng-show="!rule_id" class="btn btn-default btn-xs" ng-click="openNewScheduledReleaseChangeModal(release)">Schedule an Update</button>
-        <button ng-show="!rule_id" class="btn btn-default btn-xs" ng-click="openNewScheduledDeleteModal(release)">Schedule for Deletion</button>
+        <button ng-show="!rule_id" class="btn btn-default btn-xs" id="update-btn" ng-click="openNewScheduledReleaseChangeModal(release)">Schedule an Update</button>
+        <button ng-show="!rule_id" class="btn btn-default btn-xs" id="delete-btn" ng-click="openNewScheduledDeleteModal(release)">Schedule for Deletion</button>
```

Comment:
These colours are a little bit stark compared to the others. How about using the warning and danger classes that come with Bootstrap (http://getbootstrap.com/css/#buttons)? Those are a little dimmer, and would mean we don't have our own css to maintain either.